src/main/scala-2.12/com/linagora/gatling/imap/protocol/command/BaseIMAPCommandListener.scala

```diff
@@ -0,0 +1,38 @@
+package com.linagora.gatling.imap.protocol.command
+
+import java.util
+
+import akka.actor.ActorRef
+import com.lafaspot.imapnio.client.IMAPSession
+import com.lafaspot.imapnio.listener.IMAPCommandListener
+import com.linagora.gatling.imap.protocol.{ImapResponses, Response, UserId}
+import com.sun.mail.imap.protocol.IMAPResponse
+import com.typesafe.scalalogging.Logger
+
+import scala.collection.immutable.Seq
+
+private[command] abstract class BaseIMAPCommandListener(self: ActorRef, sender: ActorRef, userId: UserId)(logger: Logger)
```

Comment:
Well, I don't really like the "Base" naming. Why should I use this abstract class as a dev of a command listener?
What is the function it brings me?

Reply:
Yes indeed. I renamed it to RespondToActorIMAPCommandListener.
And got rid of the inheritance.